rust/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "truelayer-signing"
-version = "0.2.1"
+version = "0.2.2"
```

This should be a breaking change due to the types being changed. 
```suggestion
version = "0.3.0"
```